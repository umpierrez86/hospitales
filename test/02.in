leer_instancia
2 2
1 1
0 1
0 1
1 0
0 1
imprimir_instancia
asignar_residencias
imprimir_asignacion
Fin