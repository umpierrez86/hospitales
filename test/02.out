1>2>Cantidad hospitales: 2
Cupos hospitales: 1 1 
Cantidad estudiantes: 2
Preferencias hospitales
Preferencias hospital h0: 0 1 
Preferencias hospital h1: 0 1 
Preferencias estudiantes
Preferencias estudiante e0: 1 0 
Preferencias estudiante e1: 0 1 
3>4>(0, 1) (1, 0) 
5>Fin.
